src/INode.java

```diff
@@ -229,6 +244,17 @@ private ASTValue evalDefineProperty(WordsEnvironment environment) {
 		throw new AssertionError("Not yet implemented");	
 	}
 
+	private ASTValue evalDivide(WordsEnvironment environment) throws WordsRuntimeException {
+		ASTValue lhs = children.get(0).eval(environment).tryCoerceTo(AST.ValueType.NUM);
+		ASTValue rhs = children.get(1).eval(environment).tryCoerceTo(AST.ValueType.NUM);
+
+		if((lhs.type != AST.ValueType.NUM) || (rhs.type != AST.ValueType.NUM)) {
```

Comment:
We should try to adhere to our style guide as best we can, which means there should be a space between the "if" and the "(".